### DISCUSSION TITLE
how to add List-Unsubscribe as header

### DISCUSSION THREAD

**Discussion by The-3x** [postalserver/postal#1918] [Help with using Postal] [answered]
It will be good if i can add a list-unsubscribe header on mails so that many of the providers like gmail, outlook will treat it as a good email. also it's required to send mass mailing. is there any way to do

**Comment by willpower232** [comment, marked as answer]
this would have to be handled by whatever is sending the messages, Postal supports supplying extra headers over the API or SMTP

**Reply by marvinhinz** [reply]
I think it could be a nice, optional feature in postal itself. Just a dead simple page hosted on postal.domain.tld/unsubscribe/messageid:
"I dont want to receive emails from this sender anymore." on clicking submit the email will be added permanently to the suppression list. 

Probably a good addition for data privacy and reputation. Wdyt?

**Reply by willpower232** [reply]
Postal has no concept of mailing lists so shouldn't be responsible for handling subscribes and unsubscribes, all the data would be in  your application already which you would hope came with its own subscribe/unsubscribe process.
